modules/ROOT/pages/cloudhub-manage-props.adoc

```diff
@@ -140,6 +139,17 @@ For proprietary reasons, MuleSoft doesn't make public all internal, reserved pro
 CloudHub runtime release updates might change the reserved properties.
 --
 
+=== Non-reserved Pre-populated Properties
+
+Mule runtime pre-populates the values for the following two properties. However, you have the option to overwrite these pre-filled values. 
+
+[%header,cols="3*a"]
+|===
+|Property|Description|Example
+|`${app.name}` |The name of the Application `.jar` file. | myApplication
```

Comment:
On lines 149 and 150, I don't think the word Application should be capitalized because it is referring to an application in general. From the style guide this seems to be the case, but please use the style the Mule team requires.